pkg/cvo/sync_worker.go

```diff
@@ -99,6 +106,8 @@ type SyncWorkerStatus struct {
 
 	Actual   configv1.Release
 	Verified bool
```

Comment:
Probably drop this, since `LoadPayloadStatus.Verified` should cover it.  There shouldn't be any verification in the manifest-reconciliation loop.

Reply:
I left it in there and added a comment indicating it is the verification status of `Actual` (current) release. It ends being stored in history https://github.com/openshift/cluster-version-operator/blob/8219252a29a9e244744c87fd96a2829f0d360faf/pkg/cvo/status.go#L198 once a release has been accepted https://github.com/openshift/cluster-version-operator/blob/8219252a29a9e244744c87fd96a2829f0d360faf/pkg/cvo/sync_worker.go#L598.
`LoadPayloadStatus.Verified` is verification status of the release currently being loaded or last loaded so it could be transitory. In fact, I'm not currently using it but it seems like something that should be kept around.